staff/oriol-bover/cv/index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Oriol Bover CV</title>
+</head>
+<body>
+    <!-- Header section -->
+    <header>
+        <h1>CURRICULUM VITAE</h1>
+    </header>
+    <!-- End header section -->
+    <!-- Basic info section-->
+    <section>
+        <h2>Basic information</h2>
+        <img src="#" alt="Oriol Bover profile image">
+        <ul>
+            <li>Name: Oriol Bover Vila</li>
+            <li><address>Canigo 41 Vic (Barcelona)</address></li>
+            <li>Phone: <a href="tel:+34635524519">635524519</a></li>
+            <li>Email: <a href="mailto:bover73@gmail.com">bover73@gmail.com</a></li>
+            <li>SOCIAL NETWORK</li>
+                <ul>
+                    <li><a href="https://twitter.com/bover73" target="_blank">Twitter</a></li>
+                    <li><a href="http://github.com/boversauros/" target="_blank">Github</a></li>
+                </ul>
+        </ul>
+    </section>
+    <!-- End basic info sectioin-->
+    <hr>
+    <!-- Education section -->
+    <section>
+        <h2>Education</h2>
+        <ul>
+            <li>Advertising and public relations <strong>2012-2016</strong> <em>Not finishied</em> -- UVIC</li>
+            <li>Multimedia <strong>2017 - present </strong> -- UOC</li>
+            <li>Bootcamp javascript <strong>2018 - present </strong> -- SKYLAB CODERS ACADEMY </li>
+        </ul>
+    </section>
+    <!-- End education section -->
+    <hr>
+    <!-- xperience section -->
+    <section>
+        <h2>Experience</h2>
+        <ul>
+            <li>Private Tutor <strong>2012-2018</strong> </li>
+                <ul>
```

Comment:
nested `<ul>` and `<li>` here for just ONE item may seem TOO much. is NOT a list, then it is not necessary. same for other cases that follow this code. please, remove these unnecessary tags.